### DISCUSSION TITLE
Please add .heic for converting images

### DISCUSSION THREAD

**Discussion by SUICIDEB0YS** [Tichau/FileConverter#263] [Ideas]
Hi

I wanted to ask if it could be possible to add .heic extension for converting it to png or jpg...
Because IOS started to use it and I always need to convert it when I want to open it for example on: [https://heic.to-png.com/](https://heic.to-png.com)

Would be very nice! Thanks!

**Comment by garethsnaim** [comment]
Please, I think its about the only image format I ever need to convert, windows has changed handling, it used to be you could just drag heic images out of photos app and it would auto convert, not any more, and explorer has to have a super long think about a folder of hiec before I can see them.

Great project!

**Reply by garethsnaim** [reply]
Aw smeg, this looks like abandonware. Oh well!

**Reply by SUICIDEB0YS** [reply]
Looks like it, but I found a good website to do it: [https://heic.to-png.com/](https://heic.to-png.com/) :)
